tests/unit/StaffNotation.spec.ts

```diff
@@ -553,6 +559,11 @@ describe('StaffNotation', () => {
     expect(buildRestorationFiling).toHaveBeenCalled()
     expect(createFiling).toHaveBeenCalled()
 
+    // need to update DOM to not get previous test results
+    await Vue.nextTick()
+    await Vue.nextTick()
+    await Vue.nextTick()
```

Comment:
Hahaha...

See https://github.com/bcgov/business-create-ui/blob/main/tests/wait-for-update.ts

Reply:
Ahhhh, I see! Never knew about this 🤣 

Sev, shall I create this file here too and make it the same as Create UI or keep as is?

Reply:
I'd copy the file over. Three "await nextTick" looks really hacky, but in a loop in an external function, no one will ever know! 😆

Reply:
Thanks for pointing this out Sev! This is a pretty cool way to do it. Added the function in the latest commit.